### DISCUSSION TITLE
ZBDongle-E and Sengled bulbs

### DISCUSSION THREAD

**Discussion by aldur11** [Koenkk/zigbee2mqtt#24667] [Devices]
Does anyone have experience with ZBDongle-E and Sengled bulbs?

I've been using Z-stack-based routers/coordinators (TubesZB, TI Launchpad, ZBDongle-P) with my two Zigbee networks (around 300 Zigbee devices total) for 2+ years, and I've had nothing but troubles. I have to power cycle all of the Sengled bulbs weekly as they tend to fall off the networks or become insanely slow to respond to commands.

I recently discovered that Sengled bulbs have problems with Z-stack-based coordinators/routers. I'm considering replacing them with ZBDongle-E devices—any insight or experience on how these work together is much appreciated.

**Comment by BjornPython** [comment]
Hi, were your sengled bulbs zigbee routers? were you able to fix your network?

**Reply by aldur11** [reply]
All my Sengled bulbs are endpoints, but using the ZBDongle-E as routers and some TubesZB EFR32 MGM24 coordinators and routers fixed my problems. I haven't had a single Sengled bulb drop off the network in about a year. Such a relief to have my zigbee problems fixed.

**Reply by BjornPython** [reply]
Wow, that is awesome, especially with such a huge network with 300 zigbee devices. 
Would love to ask a few questions if that's alright.
Are you still using 2 zigbee network?
how many light bulbs do you have?
for grouping the lights, are you using zigbee2mqtt groups? or HA's?
how is the performance of the light groups in such a big network?

**Reply by aldur11** [reply]
Sure thing, I don't mind answering questions.

Yup, I'm using 2 zigbee networks.

I have around 180 bulbs.

I'm using HA groups. At first I did everything with z2m groups, but come to find out doing that many z2m groups can cause strain on the network as a broadcast gets sent to every device on the network anytime a z2m group is triggered.

Good question about performance. I expected better performance than what I'm getting, but it's not bad. Seems to take about 0.5 seconds for a light to turn on after the motion sensor detects motion. It's not bad once you get use to it.
